WcaOnRails/app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -20,7 +20,7 @@ class Api::V0::UserRolesController < Api::V0::ApiController
         if group.group_type == UserGroup.group_types[:delegate_probation]
           current_user.can_manage_delegate_probation?
         else
-          false # Don't accept any other hidden groups.
+          group.group_type == UserGroup.group_types[:translators] # Don't accept any other hidden groups.
```

Comment:
Feels like this line has a "hidden agenda". I would feel more comfortable with a switch-case statement on `group.group_type`.

Reply:
Okay, this was a mistake by me. Hidden groups are not supposed to be sent to users who doesn't have permission. If I wanted to give it to people who have permission, I should have made the group non-hidden. I've also made it to switch block.